app/uk/gov/hmrc/economiccrimelevyreturns/controllers/EstimatedEclAmountController.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyreturns.controllers
+
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.economiccrimelevyreturns.controllers.actions.{AuthorisedAction, DataRetrievalAction, ValidatedReturnAction}
+import uk.gov.hmrc.economiccrimelevyreturns.models.NormalMode
+import uk.gov.hmrc.economiccrimelevyreturns.viewmodels.checkanswers._
+import uk.gov.hmrc.economiccrimelevyreturns.views.html.EstimatedEclAmountView
+import uk.gov.hmrc.economiccrimelevyreturns.viewmodels.govuk.summarylist._
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryList
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class EstimatedEclAmountController @Inject() (
```

Comment:
```suggestion
class AmountDueController @Inject() (
```

Reply:
Should we change this regardless of the fact that the content in Confluence still states "Estimated" (I don't think Amanda has gotten round to updating this yet)? What about the Url to this... should we update that to remove "estimated" also?

Reply:
Leave the URL and content for now - but change the code I would say

Reply:
Amanda has updated the content in Confluence so I have also updated the Url to match.